objax/constants.py

```diff
@@ -21,3 +21,9 @@ class ConvPadding(enum.Enum):
     """An Enum holding the possible padding values for convolution modules."""
     SAME = 'SAME'
     VALID = 'VALID'
+
+
+class UpSample(enum.Enum):
```

Comment:
There are more methods available: https://jax.readthedocs.io/en/latest/_autosummary/jax.image.resize.html

Could you add them?

Reply:
will add them too.